package.json

```diff
@@ -22,12 +22,16 @@
     "@shopify/loom-plugin-prettier": "^2.0.1",
     "@shopify/prettier-config": "^1.1.2",
     "@shopify/typescript-configs": "^5.1.0",
+    "@types/jest": "^29.5.1",
     "eslint": "^8.55.0",
+    "eslint-plugin-prettier": "^4.2.1",
     "jest": "^29.1.0",
```

Comment:
Looks like we were at `"@types/jest": "^29.5.1",` in shopify-app-remix. Though not that, that should cause us any problems